website_sale_require_legal/readme/INSTALL.rst

```diff
@@ -0,0 +1,3 @@
+To install this module, you need to:
+
+* Install repository `OCA/website <https://github.com/OCA/website>`_.
```

Comment:
This is implicit, remove it please. (People get dependencies when downloading from apps.odoo.com)

Reply:
In PR https://github.com/OCA/e-commerce/pull/238 @chienandalu comment something like this, and @pedrobaeza answered:
		That's not true. You have to indicate dependencies outside of the current repo, which is the case.
	
	Please, check that and confirm me what i should do, thanks.